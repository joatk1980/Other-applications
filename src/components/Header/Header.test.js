import renderer from "react-test-renderer";
import Header from "./Header";

it("Renders the header", () => {
  const component = renderer.create(<Header />);
  expect(component).toMatchSnapshot();
});
