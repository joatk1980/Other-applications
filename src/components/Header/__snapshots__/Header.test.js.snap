// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the header 1`] = `
<header
  className="header"
>
  <h1>
    Local Weather App
  </h1>
</header>
`;
